app/src/main/java/org/mozilla/tv/firefox/onboarding/ReceiveTabPreboardingActivity.kt

```diff
@@ -0,0 +1,66 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.onboarding
+
+import android.app.Activity
+import android.app.Application
+import android.os.Bundle
+import android.preference.PreferenceManager
+import androidx.appcompat.app.AppCompatActivity
+import kotlinx.android.synthetic.main.receive_tab_preboarding.buttonNotNow
+import kotlinx.android.synthetic.main.receive_tab_preboarding.buttonSignIn
+import kotlinx.android.synthetic.main.receive_tab_preboarding.descriptionText
+import org.mozilla.tv.firefox.MainActivity
+import org.mozilla.tv.firefox.R
+import org.mozilla.tv.firefox.ext.serviceLocator
+
+class ReceiveTabPreboardingActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.receive_tab_preboarding)
+
+        descriptionText.text = resources.getString(
+            R.string.receive_tab_onboarding_description,
+            resources.getString(R.string.firefox_tv_brand_name_short),
+            resources.getString(R.string.firefox_tv_brand_name)
+        )
+
+        buttonSignIn.setOnClickListener {
+            application.registerActivityLifecycleCallbacks(object : Application.ActivityLifecycleCallbacks {
+                override fun onActivityResumed(activity: Activity?) {
+                    if (activity is MainActivity) {
+                        serviceLocator.fxaLoginUseCase.beginLogin(activity.supportFragmentManager)
```

Comment:
Why is it necessary to put beginLogin in `ActivityLifecycleCallbacks`? What happens if you just call the method?

Reply:
I couldn't find any better way to get a reference to the `MainActivity`, at least not any that aren't super abusable.  I'm open to better options though.

Reply:
Discussed IRL: we'll add an observable instead. It's unlikely to get abused because most of our app is in one Activity and it maybe doesn't make sense to overly obscure this code to prevent abuse if it results in a lot of added complexity and unconventional code.

Reply:
Per our conversation, replacing this with a bus.